service/internal/clientpair/clientpair.go

```diff
@@ -0,0 +1,95 @@
+package clientpair
+
+import (
+	"context"
+	"strings"
+
+	"github.com/giantswarm/apiextensions-application/api/v1alpha1"
+	"github.com/giantswarm/helmclient/v4/pkg/helmclient"
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+
+	"github.com/giantswarm/chart-operator/v2/service/controller/chart/key"
+)
+
+const (
+	// privateNamespace defines GS-protected namespace the prvHelmClient
+	// is meant for. For App CRs created outside this namespace, the
+	// pubHelmClient should be used
+	privateNamespace = "giantswarm"
```

Comment:
Is this enough? Do we use any other namespaces for our elevated charts? Just checking if there is maybe also kube-system.

Reply:
Done, no chart cr found outside the `giantswarm` namespace